mitxgraders/helpers/calc.py

```diff
@@ -363,9 +374,7 @@ def parse_algebra(self):
         function.setParseAction(self.function_parse_action)
 
         # Define parentheses
-        parentheses = Group(Suppress("(") +
-                            expr +
-                            Suppress(")"))("parentheses")
+        parentheses = Suppress("(") + expr + Suppress(")")
```

Comment:
Can we actually keep parentheses around, please? I like having these with an explicit grouping tag in debug mode.